### PR TITLE
Config: Audio sink configuration

### PR DESCRIPTION
**Pull request by MerryMage**
Added Audio tab to configuration dialog

### CODE REVIEW

File: src/citra_qt/configure_audio.cpp [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#include "audio_core/sink_details.h"
+
+#include "citra_qt/configure_audio.h"
+#include "ui_configure_audio.h"
+
+#include "core/settings.h"
+
+ConfigureAudio::ConfigureAudio(QWidget *parent) :
+    QWidget(parent),
+    ui(new Ui::ConfigureAudio) {
```
Comment (lioncash) [inline src/citra_qt/configure_audio.cpp]: std::make_unique

File: src/citra_qt/configure_audio.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace Ui {
+    class ConfigureAudio;
+}
+
+class ConfigureAudio : public QWidget {
+    Q_OBJECT
+
+public:
+    explicit ConfigureAudio(QWidget *parent = nullptr);
```
Comment (lioncash) [inline src/citra_qt/configure_audio.h]: asterisk against the type (same at the definition site)

**Comment by MerryMage** [comment]
Addressed @lioncash's comments above.

**Comment by Subv** [comment]
LGTM

**Comment by bunnei** [comment]
LGTM

**Pull request merged**